examples/term_compact.rs

```diff
@@ -0,0 +1,33 @@
+#[macro_use]
+extern crate slog;
+extern crate slog_config;
+
+use std::{io, fs};
+use std::io::Read;
+use std::path::Path;
+
+use slog::{DrainExt, Logger};
+
+
+fn read_to_string(path: &str) -> io::Result<String> {
```

Comment:
I wonder what's the best way to reuse that function. But I guess it does not have to be addressed in this PR.